aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -524,6 +500,43 @@ ICACLS C:\sites\MyWebApp /grant "IIS AppPool\DefaultAppPool":F
 
 For more information, see the [icacls](/windows-server/administration/windows-commands/icacls) topic.
 
+## HTTP/2 support
+
+::: moniker range=">= aspnetcore-2.2"
+
+[HTTP/2](https://httpwg.org/specs/rfc7540.html) is supported with ASP.NET Core in the following IIS deployment scenarios:
+
+* In-process
+  * Windows Server 2016/Windows 10 or later; IIS 10 or later
+  * Target framework: .NET Core 2.2 or later
+  * TLS 1.2 or later connection
+* Out-of-process
+  * Windows Server 2016/Windows 10 or later; IIS 10 or later
+  * Public-facing edge server connections use HTTP/2, but the reverse proxy connection to the [Kestrel server](xref:fundamentals/servers/kestrel) uses HTTP/1.1.
+  * Target framework: Not applicable to out-of-process deployments, since the HTTP/2 connection is handled entirely by IIS.
+  * TLS 1.2 or later connection
+
+For an in-process deployment when an HTTP/2 connection is established, [HttpRequest.Protocol](xref:Microsoft.AspNetCore.Http.HttpRequest.Protocol*) reports `HTTP/2`. For an out-of-process deployment when an HTTP/2 connection is established, [HttpRequest.Protocol](xref:Microsoft.AspNetCore.Http.HttpRequest.Protocol*) reports `HTTP/1.1`.
+
+For more information on the in-process and out-of-process hosting models, see the <xref:fundamentals/servers/aspnet-core-module> topic and the <xref:host-and-deploy/aspnet-core-module>.
```

Comment:
see the x article
or 
see x

either way is ok but it would be best to be consistent

Reply:
That's me ... trying to phrase it to roll off the 👅 a bit better. The titles are ...

* ASP.NET Core Module
* ASP.NET Core Module configuration reference

It would read like this in its present form ...

> For more information on the in-process and out-of-process hosting models, see the [ASP.NET Core Module](#) topic and the [ASP.NET Core Module configuration reference](#).

... but for consistency the other way it will be ...

> For more information on the in-process and out-of-process hosting models, see [ASP.NET Core Module](#) and [ASP.NET Core Module configuration reference](#).

Your call, but I like my version better. :smile: lol

Reply:
I prefer the second way you present it.  The first way is inconsistent in the way "the" goes with "reference" but "reference" is in the link text, whereas "topic" is outside of the link text.

Reply:
We were ... afaik ... always *only* linking the title. Every other word outside of the title hasn't been part of the link, except for the metapackage topic link for `App` and `All` given that those could link to the nuget.org package, so we have the word "metapackage" in the link text.

I do 99% agree and almost always .... 99% :smile: of the time ... link with "For more information, see \[link text](LINK)." ... done and done. It was just that in this case it seemed to roll off the tongue a little better.